.ci/run_nccl_test.sh

```diff
@@ -160,8 +161,9 @@ fi
     echo_hash_line
 
     MPIRUN_OPTIONS_SPECIFIC="\
-    -x NCCL_LL_THRESHOLD=1000000000 \
-    -x NCCL_TREE_THRESHOLD=1000000000 \
+    -x NCCL_PROTO=LL \
```

Comment:
Do we need test with LL128 or comma-separated list?

Reply:
CollNet algorithm in NCCL doesn't support LL128 protocol, so I'm not sure it's important at the moment. We can add test with multiple protocols/algos but we also can add some other tests. Maybe it's better to make a separate PR for the new tests. What do you think?

Reply:
Separate PR for new tests will be OK - I've submitted https://trello.com/c/W7IudX5a